var/spack/repos/builtin/packages/r-genefilter/package.py

```diff
@@ -13,6 +13,7 @@ class RGenefilter(RPackage):
     homepage = "https://bioconductor.org/packages/genefilter"
     git      = "https://git.bioconductor.org/packages/genefilter.git"
 
+    version('3.11.0', commit='a071480ffc48393a4385a94c96a752285064989f')
```

Comment:
3.11 is the version of bioconductor, I think you want 1.70.0

Reply:
Well, I don't know how to get `1.70.0`
And the relationship between `bioconductor` and `genefilter` make me confusion...

Reply:
@codeandkey can you take a look at this one?

Reply:
1.70.0 corresponds to bioconductor 3.11 (RELEASE_3_11 branch), looks like the `r-biocgenerics` was the only new dep added since the last version in the recipe.